src/iconsManifest/supportedExtensions.ts

```diff
@@ -2584,6 +2584,11 @@ export const extensions: IFileCollection = {
       extensions: ['juice'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'cmrc',
+      extensions: ['cmrc.json'],
```

Comment:
This won't work as a file extension. VS Code only takes the latest extension. It doesn't work with composed extensions.